api/types/plugin.go

```diff
@@ -183,6 +185,16 @@ func (p *PluginV1) CheckAndSetDefaults() error {
 		if len(staticCreds.Labels) == 0 {
 			return trace.BadParameter("labels must be specified")
 		}
+
+	case *PluginSpecV1_Jira:
+		if settings.Jira == nil {
+			return trace.BadParameter("missing Jira settings")
+		}
+
+		if err := settings.Jira.CheckAndSetDefaults(); err != nil {
```

Comment:
You probably should also add test to `api/types/plugin_test.go`. For example, https://github.com/gravitational/teleport/blob/621beccf9ef23fc13bd1662cc4fb320082dc3932/api/types/plugin_test.go#L112